xbmc/platform/android/activity/AndroidFeatures.cpp

```diff
@@ -1,38 +0,0 @@
-/*
- *  Copyright (C) 2012-2018 Team Kodi
- *  This file is part of Kodi - https://kodi.tv
- *
- *  SPDX-License-Identifier: GPL-2.0-or-later
- *  See LICENSES/README.md for more information.
- */
-
-#include "AndroidFeatures.h"
-
-#include "utils/log.h"
-
-#include <androidjni/JNIThreading.h>
-#include <cpu-features.h>
-
-bool CAndroidFeatures::HasNeon()
-{
-  // All ARMv8-based devices support Neon - https://developer.android.com/ndk/guides/cpu-arm-neon
-  if (android_getCpuFamily() == ANDROID_CPU_FAMILY_ARM64)
-    return true;
-
-  if (android_getCpuFamily() == ANDROID_CPU_FAMILY_ARM)
-    return ((android_getCpuFeatures() & ANDROID_CPU_ARM_FEATURE_NEON) != 0);
-
-  return false;
-}
-
-int CAndroidFeatures::GetCPUCount()
-{
-  static int count = -1;
```

Comment:
Is `android_getCpuCount()` expensive? should we continue to cache the result?

Reply:
I measured execution time of the function on my Android TV (has poor performance) and it's around 0.6 ms

Reply:
Is cpucount ever going to change during execution? I can't imagine it would. Static would be better in that case?

Reply:
Ok, better to keep the count as static